lib/eco/test/test_eco.py

```diff
@@ -33,6 +33,9 @@
 import pytest
 slow = pytest.mark.slow
 
+import logging
+#logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
In Krun we do this by passing in a command line argument. You can see the code here: